lib/tjbot.js

```
@@ -1062,9 +1062,10 @@ TJBot.prototype._normalizeColor = function(color) {
 TJBot.prototype.speak = function(message) {
     this._assertCapability('speak');
 
-    // make sure we're trying to say something
-    if (message == undefined) {
-        throw new Error("TJBot tried to speak an empty message.");
+    // make sure we're trying to say something that we have been trained to understand
+    if (message == undefined || message == '') {
+        message = 'Sorry, I did not understand';
```

I'd say we fail silently ...  return ... and print a console message.